app/imagetool.ml

```diff
@@ -119,7 +156,7 @@ let () =
   let handle_resize input_img =
     let doit x y =
       let x, y = max 1 x, max 1 y in
-      let dst = Image.create_rgb ~alpha:false x y in
+      let dst = Image.create_rgb ~alpha:true x y in
```

Comment:
Should't we only create an alpha channel if the `input_img` has one here?

Reply:
You're right, I was skipping corners.